### PR TITLE
Add Anaconda notebook to the openshift-ci pipeline

### PR DESCRIPTION
**Pull request by Xaenalt**
Related to https://github.com/opendatahub-io/notebooks/pull/28

### CODE REVIEW

File: ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml [hunk 1]
```diff
@@ -1037,6 +1064,23 @@ tests:
         requests:
           cpu: 100m
           memory: 200Mi
+    - as: jupyter-datascience-anaconda-python-3.8-test-e2e
+      cli: latest
+      commands: |
+        oc new-project jupyter-datascience-anaconda-python-3-8
+        IFS=':' read -r -a NOTEBOOK_IMAGE <<< "${JUPYTER_MINIMAL_IMAGE}"
```
Comment (atheo89) [inline ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml:1071]: Change the ${JUPYTER_MINIMAL_IMAGE} -> ${JUPYTER_ANACODA_IMAGE}. Also to line 1077

**Comment by Xaenalt** [comment]
@atheo89 out of curiosity, which prow job tests the build/deploy/etc for the Anaconda notebook so I can test it with pj-rehearse?

**Comment by atheo89** [comment]
> @atheo89 out of curiosity, which prow job tests the build/deploy/etc for the Anaconda notebook so I can test it with pj-rehearse?

I think this one will do the job(is the first one on the pj list) `pull-ci-opendatahub-io-notebooks-main-jupyter-datascience-anaconda-python-3.8-pr-image-mirror`

**Comment by Xaenalt** [comment]
@atheo89 It seems to be looking for the Anaconda notebook but failing, I'm guessing this is due to the PR not being merged, what should I do?

**Pull request merged**